docs/feature_backlight.md

```diff
@@ -30,7 +30,7 @@ You should then be able to use the keycodes below to change the backlight level.
 
 This feature is distinct from both the [RGB underglow](feature_rgblight.md) and [RGB matrix](feature_rgb_matrix.md) features as it usually allows for only a single colour per switch, though you can obviously use multiple different coloured LEDs on a keyboard.
 
-Hardware PWM is only supported on certain pins of the MCU, so if the backlighting is not connected to one of them, a software implementation will be used, and backlight breathing will not be available. Currently the supported pins are `B5`, `B6`, `B7`, and `C6`.
+Hardware PWM is only supported on certain pins of the MCU, so if the backlighting is not connected to one of them, a software PWM implementation based on a hardware timer will be used, and backlight breathing will not be available. Currently the supported pins for hardware PWM are `B5`, `B6`, `B7`, and `C6`. If audio is enabled while backlight uses software PWM, and depending on the audio pins used (Cx or Bx), the hardware timer might be either Timer 1 or Timer 3. If the audio uses both timers, then the backlight will resort to a matrix scan scheduling which might give backlight lighting artifacts (flickering).
```

Comment:
C4 and C5 are probably supported too, as they support Timer 3.

Also, it may be a good idea to clarify that the timers are only supported on certain pins, too. (like how audio lists them: https://docs.qmk.fm/#/feature_audio)

Reply:
@drashna , I've reworded completely this part of the documentation and inserted tables to show what timers are in use in which configuration of backlight and audio. I think this makes it more clear what is possible and what is not.

I double checked the code, and QMK only supports hardware PWM based backlight on B5, B6, B7 and C6. I did a quick checkin the source code and no QMK supported keyboards are using other pins on the C port besides C6. There are other backlight pin (like my xd60 which uses F5), but those don't have a hardware PWM.

I couldn't find a way to produce an html version of the documentation, so I wasn't able to see if everything renders correctly (does it use jekyll ?).

Thanks,
Brice